images/linux/scripts/installers/preimagedata.sh

```diff
@@ -19,8 +19,8 @@ cat <<EOF > $imagedata_file
     "detail": "${os_name}"
   },
   {
-    "group": "Virtual Environment",
-    "detail": "Environment: ${image_label}\nVersion: ${image_version}\nIncluded Software: ${software_url}\nImage Release: ${releaseUrl}"
+    "group": "Runner Image",
+    "detail": "Label: ${image_label}\nVersion: ${image_version}\nIncluded Software: ${software_url}\nImage Release: ${releaseUrl}"
```

Comment:
It looks like the urls included here are still referencing the old virtual-environments name
<img width="1111" alt="VE-url" src="https://user-images.githubusercontent.com/56982181/184665014-6adc02f7-7b6b-46a5-b481-ae064c63e11b.png">

Reply:
it will be visible with new image release after this PR is merged.